contract-tests/conftest.py

```diff
@@ -1,11 +1,16 @@
 # Configuration file for running contract-tests
+import os
 import pytest
 import requests
+import sys
 import urllib3
 
 # Disable any warnings about SSL connections
 urllib3.disable_warnings()
 
+# Fix our path so our tests can see modules inside Normandy
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Isn't there some better way to do this outside of python space?  I fear this will be inflexible to changes.  Let me try this locally and see if I can find something simpler.

Reply:
The reason I did this was to make sure the tests that involve signing things could see the libraries. My knowledge of Python internals is very shallow and these seemed to be a solution that worked. Definitely open to a better way of doing it